examples/huggingface/gpt-j-6b.py

```diff
@@ -130,6 +130,7 @@ def __exit__(self, type, value, traceback):
                 self.tokenizer = AutoTokenizer.from_pretrained(self.tokenizer_path)
 
 
+# NOTE: requires PIPELINE_API_TOKEN environment variable to be set
 api = PipelineCloud()
```

Comment:
Alternatively we could pass in an obvious placeholder token (no strong preference from me).

```suggestion
# Replace this token with your own
api = PipelineCloud(token="pipeline_token_value")
```

Reply:
Thanks. I've added your suggestion but also noted that they can alternatively use the env var